atlasdb-api/src/test/java/com/palantir/atlasdb/metrics/MetricPublicationArbiterTest.java

```diff
@@ -43,18 +48,18 @@ public void metricsWithNoFilterAreAccepted() {
 
     @Test
     public void metricsWithOneFilterAreFiltered() {
-        MetricPublicationArbiter arbiter = new MetricPublicationArbiter(ImmutableMap.of(
-                METRIC_NAME_1, ImmutableList.of(() -> false),
-                METRIC_NAME_2, ImmutableList.of(() -> true)));
+        MetricPublicationArbiter arbiter = createArbiter(ImmutableMap.of(
+                METRIC_NAME_1, ImmutableSet.of(() -> false),
```

Comment:
nit: Can we extract a constant for the `() -> false` and `() -> true` calls?

Reply:
Sure, will do